src/encoded/commands/write_submission_spreadsheets.py

```diff
@@ -0,0 +1,836 @@
+    if property_.pattern:
+        comment_lines.append(f"Pattern:{indent}{property_.pattern}")
+    elif is_date_format(property_.format_):
+        comment_lines.append(f"Format:{indent}YYYY-MM-DD")
+    if property_.comment:
+        comment_lines.append(f"Note:{indent}{property_.comment}")
+    if comment_lines:
+        return "\n\n".join(comment_lines)
+    return ""
+
+
+def is_date_format(format_: str) -> bool:
+    """Check if the format is a date format."""
+    return format_ == "date"
+
+
+def get_comment_height(comment_text: str) -> int:
+    """Get comment height based on number of lines.
+
+    Note: Pixels per line chosen based on trial and error.
+    """
+    lines = [line for line in comment_text.split("\n")]
+    pixels_per_line = 20
+    return len(lines) * pixels_per_line
+
+
+def get_comment_width(comment_text: str) -> int:
+    """Get comment width based on longest line.
+
+    Note: Pixels per character chosen based on trial and error.
+    """
+    lines = [line for line in comment_text.split("\n")]
+    pixels_per_char = 6
+    return max(len(line) for line in lines) * pixels_per_char
+
+
+def save_workbook(workbook: openpyxl.Workbook, file_path: Path) -> None:
+    """Save the workbook to the file path"""
+    workbook.save(filename=file_path)
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Write submission spreadsheets")
+    parser.add_argument("--output", help="Output file directory", type=dir_path)
+    parser.add_argument("--env", help="Environment", default="data")
+    parser.add_argument("--item", help="Item name", nargs="+")
+    parser.add_argument("--all", help="All items", action="store_true")
+    parser.add_argument(
+        "--workbook",
+        help="Bundle all items into a single workbook",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--separate", help="Add comments as separate cells", action="store_true"
+    )
+    parser.add_argument("--google", help="Google Sheets ID to write")
+    args = parser.parse_args()
+
+    keys = SMaHTKeyManager().get_keydict_for_env(args.env)
+    log.info(f"Found keys for {args.env}")
+    request_handler = RequestHandler(auth_key=keys)
+    if not args.output and not args.google:
+        parser.error("No output specified")
+    if args.all and args.item:
+        parser.error("Cannot specify both all and item")
+    if args.google:
+        log.info(f"Google Sheet ID: {args.google}")
+        log.info(f"Google Token Path: {GOOGLE_TOKEN_PATH}")
+        spreadsheet_client = get_google_sheet_client(args.google)
+        update_google_sheets(spreadsheet_client, request_handler)
+    elif args.all:
+        log.info("Writing all submission spreadsheets")
+        write_all_spreadsheets(
+            args.output,
+            request_handler,
+            workbook=args.workbook,
+            separate_comments=args.separate,
+            google=args.google,
```

Comment:
I think line 770 needs to be removed. Running `write_submission_spreadsheets -output ~/Downloads --env staging --workbook --all` gave an error that google wasn't an acceptable argument and commenting out this line fixed the issue.

Reply:
Ah, thanks, that was a leftover from an earlier stage of development.